modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -70,15 +71,16 @@ export function parseSignatureScript(
 ): ParsedSignatureScript | ParsedSignatureP2PKH | ParsedSignatureScript2Of3 {
   const isSegwitInput = input.witness.length > 0;
   const isNativeSegwitInput = input.script.length === 0;
```

Comment:
Drive-by observation here and I don't think this makes a difference since it's only used when `isSegwitInput` is true, but for the value to match the variable name this should be `isSegwitInput && input.script.length === 0;` Or can be renamed to `isNotScriptHashInput` but I like the other way better. Either way it makes the code a bit cleaner and might prevent a future mistake.

Reply:
agreed but will fix in follow-up